### PR TITLE
Windows and Python 3.8 support

### LINKED ISSUE #59: Windows support?
Hi, this looks like a great project!

I wanted to give it a try but after installing via `pip install git+git://github.com/tgoodlet/tractor.git`, importing tractor fails because [forkserver](https://docs.python.org/3/library/multiprocessing.html#contexts-and-start-methods) seems to not be available on my os (windows10):
```python
Python 3.7.2 (default, Feb 21 2019, 17:35:59) [MSC v.1915 64 bit (AMD64)] :: Anaconda, Inc. on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import tractor
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "C:\ProgramData\Miniconda3\envs\tractor19030502\lib\site-packages\tractor\__init__.py", line 18, in <module>
    from ._trionics import open_nursery
  File "C:\ProgramData\Miniconda3\envs\tractor19030502\lib\site-packages\tractor\_trionics.py", line 21, in <module>
    ctx = mp.get_context("forkserver")
  File "C:\ProgramData\Miniconda3\envs\tractor19030502\lib\multiprocessing\context.py", line 238, in get_context
    return super().get_context(method)
  File "C:\ProgramData\Miniconda3\envs\tractor19030502\lib\multiprocessing\context.py", line 192, in get_context
    raise ValueError('cannot find context for %r' % method) from None
ValueError: cannot find context for 'forkserver'
>>>
```
Any plan on adding windows support in the future?

### PR DESCRIPTION
**Pull request by goodboy**
This should mostly resolve #59 as well as #79 and now also #81 which gives us Python 3.8 support.

ping @chrizzFTD just to check the [new docs](https://github.com/goodboy/tractor/tree/windows_support#windows-gotchas) which detail your solution to #79.

### CODE REVIEW

**Comment by ryanhiebert** [comment]
I put my eyes on this PR, but its definitely beyond my ability to critique.

**Comment by goodboy** [comment]
Thanks @ryanhiebert for the glance!

Yeah mostly looking to make sure the docs are clear for the windows stuff.
The python 3.8 support seems to be passing in CI so I think we're good with all that hackery :smirk:

File: README.rst [hunk 1]
```diff
@@ -707,6 +699,47 @@ selected by default for speed.
...
+And execute as::
+
+    python -m application
+
+
+See `#61`_ and `#79`_ for further details.
+
+.. _freeze_support(): https://docs.python.org/3/library/multiprocessing.html#multiprocessing.freeze_support
+.. _#61: https://github.com/goodboy/tractor/pull/61#issuecomment-470053512
+.. _#79: https://github.com/goodboy/tractor/pull/79#issuecomment-470053512
```
Comment (goodboy) [inline README.rst]: shoot the `issuecomment-470053512` isn't supposed to be here..

**Comment by goodboy** [comment]
Discovered some pretty hilarious behavior with the Windows builds on travis.
Probably worth linking to [the oddities I reported in the forum](https://travis-ci.community/t/windows-python-pip-module-not-found/5480).

**Pull request merged**